google/ggrc-core
# QUICK-FIX Add issue tracker mock

⚠️ __BE PR from the FE guy, please be gentle__ 🙂

# Issue description

At the moment there's no way to fully test Issue Tracker integration on local dev environment.

# Steps to test the changes

1. Make you `docker-compose.override.yml` look similar to this
```yaml
version: '2'
services:
  cleandev:
    environment:
     <...YOUR CURRENT VARIABLES...>
     - ISSUE_TRACKER_ENABLED=true
     - ISSUE_TRACKER_MOCK=true    
```
2. Rebuild container with `./bin/containers run` ( this will just rebuild container from cached images applying new env variables. It takes few seconds )
3. Go to any Audit and enable Issue Tracker integration
4. Create assessment
5. Open Assessment info pane of the newly created assessment and Check the url of the "Ticket" link at the top of the Assessment tab. it should be "http://issue/1234"

# Solution description

Since Issue Tracker integration uses `urlfetch` API to perform requests to the integration server, we can mock these request with the `apiproxy_stub` API from the `google.appengine.api`.
The solution is based on this gist: "https://gist.github.com/kesor/1179782"

# Sanity checklist

- [ ] __N/A__ ~~My changes are covered by tests.~~

# PR Review checklist


# TODO
